File: code-recipes/java/src/test/java/messaging/SendInvoiceToBuyerTest.java
package messaging;

import util.RecipeTest;

import java.util.List;

public class SendInvoiceToBuyerTest extends RecipeTest {

    protected SendInvoiceToBuyerTest() {
        super(
                new SendInvoiceToBuyerRecipe(),
                List.of(
                    "messaging-getMessagingActionsForOrder",
                    "messaging-createUploadDestination",
                    ""
                )
        );
    }
}

File: code-recipes/php/test/messaging/SendInvoiceToBuyerTest.php
<?php

namespace Test\messaging;

use Src\messaging\SendInvoiceToBuyerRecipe;
use Test\util\RecipeTestCase;

class SendInvoiceToBuyerTest extends RecipeTestCase
{
    protected function setUp(): void
    {
        $this->recipe = new SendInvoiceToBuyerRecipe();
        $this->responses = [
            "messaging-getMessagingActionsForOrder",
            "messaging-createUploadDestination",
            "messaging-sendInvoice"
        ];
    }
}
